src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php

```diff
@@ -377,7 +387,7 @@ private static function addExtra(array &$a, \Reflector $c)
         }
     }
 
-    private static function addMap(array &$a, \Reflector $c, array $map, string $prefix = Caster::PREFIX_VIRTUAL)
+    private static function addMap(array &$a, object $c, array $map, string $prefix = Caster::PREFIX_VIRTUAL)
```

Comment:
Symfony 4.4 supports PHP 7.1, but the `object` type hint was introduced with PHP 7.2.

Reply:
There is an option to check for is_object, but given that this is a private function on a bugfix branch, and we know it is type hinted on the 5.x branch it seemed more likely to cause confusion and/or cause merge complications than provide a useful error.  Removal seemed like the best option.